<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>World of Warcraft Addon Installer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/wow_high_rez_icon_by_jocpoc-d39jgl5-120x120.png</normaloff>:/icons/wow_high_rez_icon_by_jocpoc-d39jgl5-120x120.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">QLabel { background: white; }</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="icons.qrc">:/icons/Qt_logostrap_CMYK.png</pixmap>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Depending on your internet connection, this can take some time.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowserUpdateDB"/>
        </item>
        <item>
         <widget class="QLabel" name="labelAddonsFound">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Found Addons: 0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="toolButtonMyAddonsUpdate">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/Update-icon.png</normaloff>:/icons/Update-icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonMyAddonsDelete">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/computer_key_Delete_T.png</normaloff>:/icons/computer_key_Delete_T.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBarMyAddons">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="value">
             <number>24</number>
            </property>
            <property name="format">
             <string>Searching for updates %p%</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetMyAddons">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="toolButtonInstallAddon">
            <property name="text">
             <string>Install</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/RecoveryDiskInstallIconX.png</normaloff>:/icons/RecoveryDiskInstallIconX.png</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>402</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditSearchNewAddons">
            <property name="minimumSize">
             <size>
              <width>181</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>181</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="placeholderText">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetInstallAddons">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowserDescr">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>55</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Description</string>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_5">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
   <addaction name="actionWoW_Folder"/>
   <addaction name="actionUpdate"/>
   <addaction name="actionMy_Addons"/>
   <addaction name="actionInstall_Addons"/>
   <addaction name="actionHelp"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Apps-Dialog-Shutdown-icon.png</normaloff>:/icons/Apps-Dialog-Shutdown-icon.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/database.png</normaloff>:/icons/database.png</iconset>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="toolTip">
    <string>Update the database</string>
   </property>
  </action>
  <action name="actionWoW_Folder">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/wow_high_rez_icon_by_jocpoc-d39jgl5-120x120.png</normaloff>:/icons/wow_high_rez_icon_by_jocpoc-d39jgl5-120x120.png</iconset>
   </property>
   <property name="text">
    <string>WoW Folder</string>
   </property>
   <property name="toolTip">
    <string>Set path to your WoW folder</string>
   </property>
  </action>
  <action name="actionMy_Addons">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/favorite.png</normaloff>:/icons/favorite.png</iconset>
   </property>
   <property name="text">
    <string>My Addons</string>
   </property>
   <property name="toolTip">
    <string>Installed Addons</string>
   </property>
  </action>
  <action name="actionInstall_Addons">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/favorite_add.png</normaloff>:/icons/favorite_add.png</iconset>
   </property>
   <property name="text">
    <string>Install Addons</string>
   </property>
   <property name="toolTip">
    <string>Install new addons</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/help.png</normaloff>:/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
